UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
+        BrowserUtils.resetToAboutHome(tester())
+    }
+
+
+    override func beforeAll() {
```

Comment:
That's basically what I am using this method to do. I could add the same code to the end of every test, but this does it all for me.